packages/vats/tools/protobuf-utils.js

```diff
@@ -0,0 +1,28 @@
+import { decodeBase64, encodeBase64 } from '@endo/base64';
+
+/** @import {Any} from 'cosmjs-types/google/protobuf/any.js' */
```

Comment:
I don't see `cosmjs-types` in the vats package.json. I also think we can get this from `@agoric/cosmic-proto`  / telescope, and we may prefer to import it from there